AuthenticatorBridgeKit/NSManagedObjectContext+Extension.swift

```diff
@@ -0,0 +1,52 @@
+import CoreData
+
+extension NSManagedObjectContext {
```

Comment:
🤔 With this, we'll have the same file in three different places, as near as I can tell, since there's copies of this already in PM and BWA. Does it make sense to add it again?

Reply:
This one was the one piece I felt like there wasn't a good way around duplicating it. And as you mentioned above, I was hesitant to make everything depend on the Bridge library and move this exclusively in there. 🤔